src/views/BusinessCase/Review/print.scss

```diff
@@ -0,0 +1,8 @@
+@media print {
```

Comment:
WDYT about having print-specific SCSS files? This could also just go in the component's `index.scss`.

Reply:
Doing print in the base css file makes more sense to me, but happy to have it separated, too.

Reply:
Doing print in the base css file makes more sense to me, but happy to have it separated, too.